README.rst

```diff
@@ -192,20 +192,48 @@ Scan for the device to find out its MAC address:
 
     hcitool scan
 
-The MAC address of the listed Shimmer device should end with the *BT Radio ID* imprinted on the back of the device.
+The MAC address of the listed Shimmer device should end with the *BT Radio ID* (or simply *BTID*) imprinted on the back of the device.
 Next, you can try and ping the device:
 
 .. code-block::
 
     hcitool name <mac_addr>
 
-The command should complete with the name listed previously during the scan. Now you can pair the device as follows:
+The command should complete with the name listed previously during the scan.
+
+Now verify which channel the Shimmer uses for RFCOMM:
+
+.. code-block::
+
+    sdptool records <mac_addr>
```

Comment:
Is this tool also available with the packages mentioned above? If you know which packages are needed for a Debian-based distro (Ubuntu, ...) that would be ag great add for the Readme as well.

Reply:
yes https://github.com/bluez/bluez/blob/master/tools/sdptool.rst

and I noticed your arch linux are dead, this is because they Linux folks deprecated the bluez tools..., I fixed the link in a new commit with the one that seems to be the replacement on arch (at least x86_64) https://archlinux.org/packages/extra/x86_64/bluez-deprecated-tools/